### ISSUE TITLE
ER6 - NEW PUZZLE - Mirror Code

### ISSUE THREAD

**Issue by jaspreetdhanjan** [nfynt/EscapeRoom_MR#6]
Once the player gets the mirror from the drawer, he/she can see the reflection from the mirror and use it to find the code for the lockbox. The code is written on the frame of the jigsaw puzzle.

The jigsaw puzzle should be complete prior to this and is specified in a separate ticket.
